object/host_system.go

```diff
@@ -151,3 +151,16 @@ func (h HostSystem) ExitMaintenanceMode(ctx context.Context, timeout int32) (*Ta
 
 	return NewTask(h.c, res.Returnval), nil
 }
+
+func (h HostSystem) Destroy(ctx context.Context) (*Task, error) {
```

Comment:
No need to add this method, it is already defined in the embedded `Common` struct.